pkgs/development/tools/misc/binutils/default.nix

```diff
@@ -58,10 +58,7 @@ stdenv.mkDerivation (finalAttrs: {
   pname = targetPrefix + "binutils";
   inherit version;
 
-  # HACK: Ensure that we preserve source from bootstrap binutils to not rebuild LLVM
```

Comment:
I take it the hack is no longer necessary or are we introducing an LLVM rebuild?

Reply:
The bootstrap and stdenv no longer includes GNU binutils after https://github.com/NixOS/nixpkgs/pull/322388, and evaluation will fail if it’s ever used by anything in the bootstrap. This hack isn’t necessary anymore because LLVM does not depend on GNU binutils directly or indirectly on Darwin.

One can verify that LLVM is not rebuild by doing the following. The first LLVM is built during the xclang stage with the bootstrap tools. The second one is built in stage 3.
```shell
$ nix-store -qR $(nix-instantiate . -A binutils) | grep llvm-16.0.6.drv
/nix/store/r7gx4bh0wh0bvcxn6i3n2d6b0z7sbm3d-llvm-16.0.6.drv
/nix/store/hfsyv59k1y0b1f9wmfjav08svwf3dk7c-llvm-16.0.6.drv
```

Reply:
Note that llvm-src is referenced three times. The third time is when building the manpages for darwin.binutils.